templates/salesorderlines.json

```diff
@@ -86,6 +86,41 @@
       "type": "dataset"
     },
     "transform": [
+      {
+        "rules": {
+          "default": [
+            [
+              "copy",
+              "*"
+            ],
+            [
+              "add",
+              "$payload",
+              [
+                "apply",
+                "filter-payload",
+                "_S."
+              ]
+            ]
+          ],
+          "filter-payload": [
```

Comment:
This shouldn't need to be in this connector. The template should be filtering _ and $ prefixed variables and it doesn't look like you use documentId earlier in the pipe so that can be marked as readonly in the mapping (if it is mapped at all). Because only mapped properties make it to the share pipe that will accomplish the same goal. If the template isn't filtering _ and $ prefixed variables then that is a bug in the template.